### PR TITLE
Fix service etc to make Mayan work on k8s

### PR DESCRIPTION
**Pull request by rndsrc**

### CODE REVIEW

File: db-secret.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  # Secret for Mayan EDMS' database
+  name: db-secret
+data:
+  # Data must use base64 encode; try `echo -n '****' | base64`
+  password: bWF5YW5kYnBhc3M=
```
Comment (matthewturk) [inline db-secret.yaml:14]: Is it OK to commit this?

Comment (rndsrc) [reply db-secret.yaml:14]: This is just the sample password from the docker compose example.  I commented in the `README.md` in real deployment people should change the password and *not* commit their password.

Comment (matthewturk) [reply db-secret.yaml:14]: So what you're *saying* is that I should have read the entire set of files before making an ill-informed question?  ;-)

Comment (rndsrc) [reply db-secret.yaml:14]: I updated the comments so it's more explicitly that the password should be changed in real deployment. ;-)  Alternative, we may remove the secret.yaml files and provide commands for people to manually create secret.

**Pull request merged**